1.0/learn/by-example/http-1-1-to-2-0-protocol-switch.html

```diff
@@ -2,10 +2,10 @@
 layout: ballerina-example-page
 title: HTTP 1.1 to 2.0 Protocol Switch
 description: Ballerina by Example is a hands-on introduction to Ballerina using annotated example programs. Check out the first example or browse the full list below.
-permalink: /1.1/learn/by-example/http-1.1-to-2.0-protocol-switch
+permalink: /1.0/learn/by-example/http-1-1-to-2-0-protocol-switch
 redirect_from:
-  - /v1-1/learn/by-example/http-1.1-to-2.0-protocol-switch
-  - /v1-1/learn/by-example/http-1.1-to-2.0-protocol-switch.html
+  - /v1-0/learn/by-example/http-1-1-to-2-0-protocol-switch
```

Comment:
Do we need to update the ballerina version here also?

Reply:
Fixed.